chaos-bugbounty-list.json

```diff
@@ -1752,6 +1752,26 @@
          "domains":[
             "pon-cat.com"
          ]
+      },
+      {
+         "name": "Playstation",
+         "url": "https://hackerone.com/playstation",
+         "bounty": true,
+         "domains":[
+            "playstation.net",
+            "sonyentertainmentnetwork.com",
+            "api.playstation.com",
```

Comment:
Please remove `api.playstation.com` as it only requires domain name as input which is already in the list.

Reply:
done